include/swift/AST/Builtins.def

```diff
@@ -1004,6 +1004,11 @@ BUILTIN_MISC_OPERATION_WITH_SILGEN(AutoDiffAllocateSubcontextWithType, "autoDiff
 BUILTIN_MISC_OPERATION_WITH_SILGEN(BuildOrdinarySerialExecutorRef,
                                    "buildOrdinarySerialExecutorRef", "n", Special)
 
+/// Build a Builtin.Executor value from an "ordinary" task executor
+/// reference.
+BUILTIN_MISC_OPERATION_WITH_SILGEN(BuildOrdinaryTaskExecutorRef,
+                                   "buildOrdinaryTaskExecutorRef", "n", Special)
+
```

Comment:
I don't know that you need to copy this stuff about builtins.  Let's wait until we have a need to build task executors into the compiler.

Reply:
Okey yeah we can add when necessary, thanks!